clients/mediaconvert.go

```diff
@@ -290,23 +291,43 @@ func getFileHTTP(ctx context.Context, url string) (io.ReadCloser, error) {
 	return resp.Body, nil
 }
 
-func copyFile(ctx context.Context, sourceURL, destOSBaseURL, filename string) error {
+func copyFile(ctx context.Context, sourceURL, destOSBaseURL, filename string) (*int64, error) {
 	ctx, cancel := context.WithTimeout(ctx, 10*time.Minute)
 	defer cancel()
 	content, err := getFile(ctx, sourceURL)
 	if err != nil {
-		return fmt.Errorf("download error: %w", err)
+		return nil, fmt.Errorf("download error: %w", err)
 	}
 	defer content.Close()
 
 	err = UploadToOSURL(destOSBaseURL, filename, content, 1*time.Minute)
 	if err != nil {
-		return fmt.Errorf("upload error: %w", err)
+		return nil, fmt.Errorf("upload error: %w", err)
 	}
-	return nil
+
+	// reread and early close session to uploaded file to OS
+	// as follow-up, make
+	storageDriver, err := drivers.ParseOSURL(destOSBaseURL, true)
+	if err != nil {
+		return nil, err
+	}
+	sess := storageDriver.NewSession("")
+	info, err := sess.ReadData(context.Background(), filename)
+	if err != nil {
+		return nil, err
+	}
+
+	defer info.Body.Close()
+	defer sess.EndSession()
+
+	if err != nil {
+		return nil, err
+	}
+
+	return info.FileInfo.Size, nil
```

Comment:
If this is all so that we can get the size, could we have the OS return this instead?